### PR TITLE
lv_freetype and enable the cache manger

### PR DESCRIPTION
**Pull request by as564335sa**
This is the `lv_freetype` used in my project,it enable the cache manger use `USE_CAHCE_MANGER`；
It is very fast when cache manger is enabled, which is almost the same as the built-in Font;
I'm not very familiar with freetype. If it have any problems, please help me to modify it;

### CODE REVIEW

File: lv_freetype.c [hunk 1]
```diff
@@ -1,193 +1,188 @@
-/**
- * @file lv_freetype.c
- *
- */
-
-/*********************
```
Comment (kisvegabor) [inline lv_freetype.c:6]: Please keep these sections.

**Comment by as564335sa** [comment]
> Hi,
> Thank you! I found a few things, could you fix them?

I have fix it and push them

**Comment by kisvegabor** [comment]
Looks good thank you!

I'll merge it later today when v7 is released.

**Comment by as564335sa** [comment]
> Looks good thank you!
> 
> I'll merge it later today when v7 is released.

Hi,
I used it very well before dev-7. Today i updated the V7 release. but the Chinese display is not complete. I don't know what's wrong. Can you test it?

**Pull request merged**